Python/pip.md

```diff
@@ -4,12 +4,16 @@ macOS comes with Python so there's a chance `pip` is already installed on your m
 
 ## Installation
 
-    curl https://bootstrap.pypa.io/get-pip.py > get-pip.py
-    sudo python get-pip.py
+```console
+curl https://bootstrap.pypa.io/get-pip.py > get-pip.py
+sudo python get-pip.py
+```
 
 To verify `pip` is installed properly run
 
-    pip --version
+```console
+$ pip --version
```

Comment:
This `$` is not required either. We should really look into automating these things.

Reply:
same as above